lightkurve/mast.py

```diff
@@ -61,86 +69,242 @@ def search_kepler_products(target):
             target_name = 'ktwo{:09d}'.format(target)
         else:
             raise ValueError("Not in the Kepler KIC or EPIC ID range")
-        obs = Observations.query_criteria(target_name=target_name,
+        obs = Observations.query_criteria(objectname=target_name,
```

Comment:
Is there any point left in having a separate query for targets that look like EPIC/KIC IDs and those that don't?  I wonder if we can get rid of all this and just run the call to `query_criteria` that gets executed beneath the `except ValueError:` block right now?  What do you think?

Reply:
I think this will probably work. Let's try it.